sys/congure/abe/Kconfig.config

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2021 Freie Universität
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+# XXX This file only is required since there is no easy way to use these config
+# options with both the final MODULE_SHELL and KCONFIG_USEMODULE_SHELL in a
+# nicely looking and easy to migrate way. After migration, the content of this
+# file can be folded back into `sys/shell/Kconfig`
```

Comment:
How is this module related to the shell module?

Reply:
I think this is copy-pasta from `congure_test`. Will remove.

Reply:
Nope, the text was still important to be here, just the module names were wrong. Replaced `SHELL` with `CONGURE_ABE` and `shell/` with `congure/abe/` respectively.